src/github/graphql.ts

```diff
@@ -140,6 +140,7 @@ export interface TimelineEventsResponse {
 			}
 		}
 	};
+	rateLimit: RateLimit;
```

Comment:
We don't seem to be doing anything with this?

Reply:
I emit this out in debug log as both @RMacfarlane and me ran into Rate Limit once this week, we'd like to know this info while debugging.

Reply:
Ah, ok.